packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.test.ts

```diff
@@ -56,23 +56,50 @@ async function visualizeChildData(data: unknown): Promise<VisualChildNode> {
 }
 
 describe("DefaultVisualizers unit tests", () => {
-	it("SharedCell", async () => {
+	it("SharedCell (Value Node)", async () => {
```

Comment:
Nit: Rather than expressing the test names in terms of the expected function **output**, I would recommend naming them in terms of the input conditions. E,g, `(primitive data)` and `(json)`